src/utils/masonryDetails.js

```diff
@@ -1,97 +1,51 @@
-const masonryDetails = {
-  accordion: {
-    'Focus Expectations': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Semantic Elements': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Aria Attributes': {
-      rowStart: 'auto',
-      numberOfRows: 1,
-    },
-    'Keyboard Interactions': {
-      rowStart: 2,
-      numberOfRows: 1,
-    },
-    'Additional Resources': {
-      rowStart: 2,
-      numberOfRows: 1,
-    },
+import { marked } from 'marked';
+
+export const rowStart = (markdown) => {
+  const htmlFromMd = marked.parse(markdown.replace(/\\/g, ''), {
+    breaks: true,
+  });
+
+  switch (true) {
+    case htmlFromMd.length < 100:
+      return 1;
+
+    case htmlFromMd.length > 100 && htmlFromMd.length < 300:
+      return 2;
+
+    case htmlFromMd.length > 300:
+      return 3;
+    default:
+      return 1;
+  }
+};
+
+const specificationBlocks = {
+  'Focus Expectations': {
+    rowStart,
+    // numberOfRows: 1,
   },
-  dialog: {
-    'Keyboard Interactions': {
-      rowStart: 1,
-      numberOfRows: 2,
-    },
-    'Focus Expectations': {
-      rowStart: 3,
-      numberOfRows: 1,
-      columnStart: 2,
-    },
-    'Semantic Elements': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Aria Attributes': {
-      rowStart: 3,
-      numberOfRows: 1,
-      columnStart: 1,
-    },
-    'Additional Resources': {
-      rowStart: 2,
-      numberOfRows: 1,
-    },
+  'Semantic Elements': {
+    rowStart,
+    // numberOfRows: 1,
+    // columnStart: 2,
   },
-  disclosure: {
-    'Focus Expectations': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Semantic Elements': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Aria Attributes': {
-      rowStart: 3,
-      numberOfRows: 1,
-      columnStart: 1,
-    },
-    'Keyboard Interactions': {
-      rowStart: 2,
-      numberOfColumns: 2,
-    },
-    'Additional Resources': {
-      rowStart: 3,
-      numberOfRows: 1,
-      columnStart: 1,
-    },
+  'Aria Attributes': {
+    rowStart,
+    // numberOfRows: 1,
```

Comment:
Did you want to keep these comments in? Or are these from the hard-coded version?

Reply:
These will come out before the PR is done. I am still thinking that Monday I'll work on making columnStart and numberOfRows dynamic as well.